01-intro.Rmd

```diff
@@ -0,0 +1,127 @@
+
+* `git config --global user.email <NHS email address>`
+* `git config --global user.name <your name>`
+
+At any time you can check what the current user details are using:
+
+* `git config --global user.email`
+
+
+### GitHub {-}
+
+These instructions are for first-time users of GitHub and highlight the steps in setting up a GitHub account.
+
+1. **Sign up** - In order to use GitHub, you'll need to have a personal GitHub account, you can set that up on the [GitHub website](http://github.com). If using your NHS email address ensure you **don't use the same password as your NHS email account**. 
+2. **Edit your details** - you will be able to set these items up as part of the sign up process. Otherwise, follow the links and edit them now.
+
+* Go to your [Profile Settings](https://github.com/settings/profile) and add your details with a picture of yourself if you feel comfortable to do so.
+* Go to your [Account Settings](https://github.com/settings/admin) and make sure your username is set to be your full name. This helps to identify users as generated GitHub handles tend to pretty obscure.
+
+3. **Get added to Organisations** - this step is optional.
+
+* If you're doing work within the Transforming Publishing team. Email one of the admins [here](https://github.com/orgs/NHS-NSS-transforming-publications/teams/transforming-publishing-team/members?utf8=%E2%9C%93&query=+role%3Amaintainer) to request being added to the organisation.
+* If you work within Public Health & Intelligence (PHI), the [Health & Social Care Scotland](https://github.com/Health-SocialCare-Scotland) GitHub may be of interest. Repositories can be requested and set up for team working.
+
+
+### RStudio {-}
+
+First [R](https://www.r-project.org/) and [RStudio](https://www.rstudio.com/) must be installed on your machine. Just as with git, if you work within NHS National Services Scotland these can be requested through the [Service Portal](https://nhsnss.service-now.com/sp). Once authorised, this will allow you to download it from the Software Center on your machine. You may also be able to use the server versions of RStudio and the project workflow will be the same.
+
+1. **Once in RStudio, go to Tools in the menu bar > Global Options > Git/SVN**:
+
+```{r fig.align='center', echo=FALSE, include=identical(knitr:::pandoc_to(), 'html')}
+knitr::include_graphics('images/rstudio-globalopt-gitsvn.png', dpi = NA)
+```
+&nbsp;
+
+2. **If "Git executable:" shows "*(none)*", click "Browse" and select the git executable installed on your system.** - On Windows, git.exe will likely be be in "Program Files", specifically try `C:/Program Files/Git/bin/git.exe`. That's it for attaching RStudio and Git. This will allow you to utilise the RStudio Git GUI tools inside R Projects. 
```

Comment:
Occasionally, there is a filepath on the server, but it points to the wrong folder. I would highlight what the filepath should be for the server (/usr/bin/git) and then for the desktop just say the filepath should point to the .../bin/git.exe file in the folder git was installed to. This is quite dependent on the user and their OS so I wouldn't provide an example as that might confuse people where it is different. Might be worth pointing them towards somewhere they can get help if they can't find it, though.